husky_directory/static/js/setSearchNameAttribute.js

```diff
@@ -0,0 +1,7 @@
+let searchformfield = document.getElementById("name");
+document.getElementById("method").onchange = changeListener;
+
+function changeListener(){
+    var value = this.value
+    searchformfield.setAttribute("name", value);
+}
```

Comment:
Oh, I approved your PR but I'd appreciate some more info here...

Reply:
I'll put a comment at the top. 
The purpose of this script is to set the selected (dropdown) value to whichever the user selected when they ran the query. This will maintain parity with the existing directory.
Before this code, the dropdown would always be set to "Name" on the result set even if the user selected another option like "Phone".
Has no affect on the search results.

Reply:
I suppose now that I understand it better, I would have requested some changes:

1. The element id shoudn't be "name" if it's not always a name. It should, "search-term" or "query" or something. Similarly, I find the attribute "method" equally confusing, as usually when we talk about methods in API calls, we talk about GET, POST, PUT, etc. "query-attribute" "query-value" would make more sense to me as DOM element ids. (Or something like those, doesn't have to be those exactly.)
2. (just noticed) `searchformfield` should be `searchFormField`.
3. However I don't think JS is the right solution here. We render the query itself using jinja, we should just conditionally render the `selected` attribute on the `option` if it matches `serach_result['query'].keys()|first`

I think in general, this list of options should be rendered from the actual input model rather than hard-coded, but I also think that's out of scope for the current task. 

So, for now, if you could address points #1 and #2 that would be great. I know how to make #3 happen. If we're going to use jinja we should embrace it and make use of these pydantic models that do a lot of this heavy lifting for us.

Reply:
(ignore those # links; github is trying to be cute but it's wrong this time)

Reply:
(to avoid ambiguity, none of this is a blocker, but I do wish I'd have caught nubmers 1 and 2 before approving :))

Reply:
Thanks for the constructive feedback.
1. Agreed, the id should be something other than `name`. I'll make that change. `query` sounds good.
2. I'll take care of that.
3. My approach was to use jinja only but after trying to find something with only jinja (maybe it was right under my nose), I went ahead and used JS. Please do make the change if you'd like :)